blocks/link-list/link-list.css

```diff
@@ -0,0 +1,69 @@
+.link-list {
+  --color-text: light-dark(#191919, #f2f2f2);
+  --color-border: light-dark(#c6c6c6, #505050);
+  --icon-size: 20px;
```

Comment:
can we use rem? you might need to take the height/width off the icons themselves